### PR TITLE
 #184760048 Edit user profile

### PR DESCRIPTION
**Pull request by UwaweTera**
### **What does this PR do?**

Edit user profile

### **Description of task to be completed ?**
Once user is logged he/she can update his/her Profile like change billing Address, profile picture and more  user data

### **How should it be tested manually**
clone it and do: `npm start`
after that login: `/login`
once you login you can update your profile by click on profile Update button
### **Any background context you want to provide ?**
N/A

### Screenshots (if appropriate)
profile1
profile2

### **what are the relevant pivotal tracker stories ?**

https://www.pivotaltracker.com/n/projects/2628461/stories/184760048

### CODE REVIEW

**Review by Ntarecedrick — CHANGES_REQUESTED** [review]
can you squash your commits they are five

File: src/redux/features/actions/updateProfile.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import instance from "../../axiosinstance";
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+const token =  localStorage.getItem("token");
+// const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7Im5hbWUiOiJBZG1pbiBUcm9qYW4iLCJlbWFpbCI6ImFkbWluMTIzQGdtYWlsLmNvbSIsImlkIjoxLCJyb2xlIjoiYWRtaW4iLCJzdGF0dXMiOiJhY3RpdmUiLCJwcm9maWxlUGljIjoiaHR0cHM6Ly9yZXMuY2xvdWRpbmFyeS5jb20vZG1qeHVreDA5L2ltYWdlL3VwbG9hZC92MTY3NTg0NDY5Mi9wcm9maWxlcy9Qcm9maWxlLUF2YXRhci1QTkctRnJlZS1Eb3dubG9hZF9wYXFmcmYucG5nIiwibGFzdFRpbWVQYXNzd29yZFVwZGF0ZWQiOiIyMDIzLTAzLTE3VDE0OjAzOjE0LjQwMFoifSwiaWF0IjoxNjgwODUwOTIwfQ.2URytQCe6jh5NtDBXWqRdzdr7IFFu87c5_uPrZbGPR4";
```
Comment (Ntarecedrick) [inline src/redux/features/actions/updateProfile.js]: can you remove this comment since it's not being used

**Review by Ntarecedrick — CHANGES_REQUESTED** [review]

**Review by Ntarecedrick — CHANGES_REQUESTED** [review]
on the deployed link the icons seems to be broken maybe you can change that because all the website icons aren't working

File: src/redux/features/actions/updateProfile.js (hunk 1 above)
Comment (UwaweTera) [reply src/redux/features/actions/updateProfile.js]: solved my friend @Ntarecedrick

**Review by Ntarecedrick — APPROVED** [review]

**Review by Eli250 — APPROVED** [review]

**Pull request merged**